src/ViewHelpers/ArgumentViewHelper.php

```diff
@@ -0,0 +1,58 @@
+<?php
+declare(strict_types=1);
+namespace TYPO3Fluid\Fluid\ViewHelpers;
+
+/*
+ * This file belongs to the package "TYPO3 Fluid".
+ * See LICENSE.txt that was shipped with this package.
+ */
+
+use TYPO3Fluid\Fluid\Component\EmbeddedComponentInterface;
+use TYPO3Fluid\Fluid\Component\SequencingComponentInterface;
+use TYPO3Fluid\Fluid\Core\Parser\Sequencer;
+use TYPO3Fluid\Fluid\Core\Rendering\RenderingContextInterface;
+use TYPO3Fluid\Fluid\Core\ViewHelper\AbstractViewHelper;
+
+/**
+ * Argument assigning ViewHelper
+ *
+ * Assigns an argument for a parent ViewHelper call when
```

Comment:
Drop "when the parent ViewHelper supports it." because every ViewHelper will support this method.

I would also reword to "Assigns an argument to the parent node" since this works for every node that supports arguments (Atom, ViewHelper).